wix-jsdoc-babel/src/transform.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    presets: ['@babel/preset-env']
-}
\ No newline at end of file
+  presets: ['@babel/preset-env', '@babel/preset-typescript']
```

Comment:
does the order of presets matter? (maybe typescript should be first)

Reply:
https://babeljs.io/docs/en/presets#preset-ordering it will run typescript first, because babel run them from last to first